beacon_node/http_api/Cargo.toml

```diff
@@ -30,5 +30,5 @@ slot_clock = { path = "../../common/slot_clock" }
 [dev-dependencies]
 store = { path = "../store" }
 environment = { path = "../../lighthouse/environment" }
-tree_hash = { path = "../../consensus/tree_hash" }
-discv5 = { version = "0.1.0-alpha.10", features = ["libp2p"] }
+tree_hash = "0.1.0"
```

Comment:
Looks like the `tree_hash` in the code base is 0.1.1, not sure what the difference here is though.